wurst/lib/Pool.wurst

```diff
@@ -10,6 +10,11 @@ public class Pool<T>
         totalWeight += weight
         options.put(t, weight)
 
+    function add(T t, int weight)
+        let realWeight = weight.toReal()
+        totalWeight += realWeight
+        options.put(t, realWeight)
+

```

Comment:
Perhaps we should just switch all of the weights to `int` though I think the random number generator may be better for reals.

Reply:
The function wont accept the int globals as args without having the ugly .toReal() everywhere unless there is another constructor for int params.

Reply:
Ok sounds good. Perhaps we can just make it call `add` with the real like
```
function add(T t, int weight)
    add(t, weight.toReal())
```